systest/Makefile

```diff
@@ -7,7 +7,7 @@ clusters ?= 1
 size ?= 10
 level ?= debug
 bootstrap ?= 5m
-node_selector ?= cloud.google.com/gke-nodepool=dmitry-ci-preemtible-1
```

Comment:
your branch is not the latest version. i have updated this a while back.

Reply:
Fixing...